src/tactic/linarith.lean

```diff
@@ -134,7 +231,7 @@ str determines the direction of the comparison -- is it < 0, ≤ 0, or = 0?
 @[derive inhabited]
 meta structure comp :=
```

Comment:
```suggestion
meta structure comp : Type :=
```
The doc string needs to be updated too since `coeffs.keys` no longer exists, right?